lib/graphql/schema.graphql

```diff
@@ -1809,21 +1824,6 @@ type Mutation {
     """
     id: Int!
   ): NotificationType
-
-  """
-  Endpoint to accept or reject an invitation to become a member
-  """
-  replyToMemberInvitation(
-    """
-    The ID of the invitation to accept or decline
```

Comment:
@Betree  you know why these are out?

Reply:
You probably updated API's schemas against an old version of the API. Updating API to latest and rebasing this PR on master should help you to get a clean state.